modules/mono/glue/GodotSharp/GodotSharp/Core/Bridge/GCHandleBridge.cs

```diff
@@ -18,5 +18,30 @@ internal static void FreeGCHandle(IntPtr gcHandlePtr)
                 ExceptionUtils.LogException(e);
             }
         }
+
+        [UnmanagedCallersOnly]
+        internal static godot_bool GCHandleIsTargetCollectible(IntPtr gcHandlePtr)
+        {
+            try
+            {
+                var target = GCHandle.FromIntPtr(gcHandlePtr).Target;
+
+                if (target is null)
+                    return godot_bool.True;
```

Comment:
Here you return `true` when the target is null, but if the target is a `Delegate` the method `DelegateUtils.IsDelegateCollectible` returns `false` if the delegate is null. Obviously if the delegate is null it won't every reach the `DelegateUtils.IsDelegateCollectible` method invocation since it will return here. I was wondering why the 2 methods treat null differently.

Reply:
TBH this should be unreachable anyways, as it means that the passed handle was already released / collected already, also these checks are really only there to ignore things from the GodotTools assembly and the check generally should try to reload things to be on the safe side.

Reply:
> TBH this should be unreachable anyways, as it means that the passed handle was already released / collected already

That's what I was thinking, but I figure it's safe to return `true` in this case. What concerns me is why this similar case returns `false` in:

https://github.com/RedworkDE/godot/blob/f4e115668625fcbd543dca293ecf48c402904b64/modules/mono/glue/GodotSharp/GodotSharp/Core/DelegateUtils.cs#L550-L553

Shouldn't it also return `true`? Although I realize I'm being nit-picky.